lib/reporting.rb

```diff
@@ -94,9 +94,9 @@ def modules
 
 private
 
-  # @return [Integer] Number of users who have added at least one note
+  # @return [Integer] Number of users who have added at least one note, given the note body is not empty
   def get_users_with_notes_count
-    User.joins(:notes).distinct.count
+    User.joins(:notes).distinct.count { |user| user.notes.any? { |note| !note.body.strip.empty? } }
```

Comment:
I feel like there maybe value in turning this into a scope on the `User` model and would perhaps refactor into something cleaner if we added a utility predicate to the `Note` model. @jack-coggin play around with that idea and see what you think. I'll leave the review of whether this meets @ebrett's requirements to him.

Reply:
Is this the sort of thing you had in mind? https://github.com/DFE-Digital/early-years-foundation-recovery/pull/622/commits/4abd01bb64dc1d58a1d22802552d10ed42f4630a